tests/mssql-tests.py

```diff
@@ -32,6 +32,8 @@ def parse_options():
     parser.add_option('--azure', action='store_true', default=False,
                       help='If present, will connect as Azure, otherwise as '
                            'standard MSSQL')
+    parser.add_option('--tds_version', action="store", default="7.1",
+                      help='Specifies th TDS protocol version')
```

Comment:
IIRC the `default` parameter doesn't show when you run `--help`, so you will need to specify it at `help` parameter:
```
parser.add_option('--tds_version', action="store", default="7.1",
                              help='Specifies th TDS protocol version ("7.1" by default)')
```